menas/src/main/scala/za/co/absa/enceladus/menas/HDFSConfig.scala

```diff
@@ -28,7 +28,7 @@ import org.springframework.context.annotation.{Bean, Configuration}
 class HDFSConfig @Autowired() (spark: SparkSession) {
   private val logger = LoggerFactory.getLogger(this.getClass)
 
-  @Value("${menas.hadoop.auth.method}")
+  @Value("${menas.hadoop.auth.method:}")
```

Comment:
It used to fail before because of this, but we solved it another way. Anyway, it seems to be the only one without a colon here

Reply:
Let's not forget what the colon means - in general, it changes the meaning of the property `${"my.property.key"}` (without default value - so failing to find it in the config should cause a failure) to `${"my.property.key:defaultValueForThatKey"}` (where failing to find the key in config just fallbacks to the default - and the _nothing_ after the colon means default empty string, but having default provided nonetheless)

Most probably, you all knew all this, just stating it so we just "don't add" colons because it is present elsewhere, but we consider the true meaning - and here that it should be _whether it is reasonable to provide default empty string when the key is not found in properties_.